admin/model_group_alerts_config.go

```diff
@@ -30,9 +30,9 @@ type GroupAlertsConfig struct {
 	Notifications *[]AlertsNotificationRootForGroup `json:"notifications,omitempty"`
 	// Date and time when someone last updated this alert configuration. This parameter expresses its value in the ISO 8601 timestamp format in UTC.
 	// Read only field.
-	Updated         *time.Time                  `json:"updated,omitempty"`
-	MetricThreshold *FlexClusterMetricThreshold `json:"metricThreshold,omitempty"`
-	Threshold       *GreaterThanRawThreshold    `json:"threshold,omitempty"`
+	Updated         *time.Time                      `json:"updated,omitempty"`
+	MetricThreshold *FlexClusterMetricThreshold     `json:"metricThreshold,omitempty"`
+	Threshold       *StreamProcessorMetricThreshold `json:"threshold,omitempty"`
```

Comment:
inner property `Threshold` in now `*float64` over `*int` due to this change, this appears to be spec correction.